doc/conf.py

```diff
@@ -68,7 +68,7 @@
 source_suffix = '.rst'
 
 # The encoding of source files.
-#source_encoding = 'utf-8'
+# source_encoding = 'utf-8'
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.